features/routing/abrouting.feature

```diff
@@ -11,28 +11,21 @@ Feature: Testing abrouting
 
     Given I switch to the first user
     And I have a project
-    Given I obtain test data file "routing/abrouting/caddy-docker.json"
+    Given I obtain test data file "routing/abrouting/abtest-websrv1.yaml"
     When I run the :create client command with:
-      | f | caddy-docker.json |
+      | f | abtest-websrv1.yaml |
     Then the step should succeed
-    And the pod named "caddy-docker" becomes ready
+    And a pod becomes ready with labels:
+      | name=abtest-websrv1 |
     And evaluation of `pod.ip` is stored in the :pod_ip1 clipboard
-    Given I obtain test data file "routing/abrouting/caddy-docker-2.json"
+    Given I obtain test data file "routing/abrouting/abtest-websrv2.yaml"
     When I run the :create client command with:
-      | f | caddy-docker-2.json |
+      | f | abtest-websrv2.yaml |
     Then the step should succeed
-    And the pod named "caddy-docker-2" becomes ready
+    And a pod becomes ready with labels:
+      | name=abtest-websrv2 |
     And evaluation of `pod.ip` is stored in the :pod_ip2 clipboard
-    Given I obtain test data file "routing/abrouting/unseucre/service_unsecure.json"
-    When I run the :create client command with:
-      | f | service_unsecure.json |
-    Then the step should succeed
-    Given I obtain test data file "routing/abrouting/unseucre/service_unsecure-2.json"
-    When I run the :create client command with:
-      | f | service_unsecure-2.json |
-    Then the step should succeed
-    Given I wait for the "service-unsecure" service to become ready
-    Given I wait for the "service-unsecure-2" service to become ready
```

Comment:
the new abtest-websrv[12].yaml already created the services, above steps can be removed.